src/Text/Markdown/SlamDown/Parser/Inline.purs

```diff
@@ -291,30 +290,36 @@ inlines = L.many inline2 <* PS.eof
               pure $ Right $ SD.TextBox $ SD.transTextBox (M.Just >>> Compose) def
             M.Nothing →
               pure $ Left case template of
-                SD.DateTime _ →
+                SD.DateTime SD.Minutes _ →
                   "Incorrect datetime default value, please use \"YYYY-MM-DD HH:mm\" or \"YYYY-MM-DDTHH:mm\" format"
+                SD.DateTime SD.Seconds _ →
+                  "Incorrect datetime default value, please use \"YYYY-MM-DD HH:mm:ss\" or \"YYYY-MM-DDTHH:mm:ss\" format"
                 SD.Date _ →
                   "Incorrect date default value, please use \"YYYY-MM-DD\" format"
-                SD.Time _ →
-                  "Incorrect time default value, please use \"HH:mm\" or \"HH:mm:ss\" format"
+                SD.Time SD.Minutes _ →
+                  "Incorrect time default value, please use \"HH:mm\" format"
+                SD.Time SD.Seconds _ →
+                  "Incorrect time default value, please use \"HH:mm:ss\" format"
                 SD.Numeric _ →
                   "Incorrect numeric default value"
                 SD.PlainText _ →
                   "Incorrect default value"
 
     parseTextBoxTemplate ∷ P.Parser String (SD.TextBox (Const Unit))
     parseTextBoxTemplate =
-      SD.DateTime (Const unit) <$ PC.try parseDateTimeTemplate
+      SD.DateTime SD.Seconds (Const unit) <$ PC.try (parseDateTimeTemplate SD.Seconds)
+        <|> SD.DateTime SD.Minutes (Const unit) <$ PC.try (parseDateTimeTemplate SD.Minutes)
         <|> SD.Date (Const unit) <$ PC.try parseDateTemplate
-        <|> SD.Time (Const unit) <$ PC.try parseTimeTemplate
+        <|> SD.Time SD.Seconds (Const unit) <$ PC.try (parseTimeTemplate SD.Seconds)
+        <|> SD.Time SD.Minutes (Const unit) <$ PC.try (parseTimeTemplate SD.Minutes)
         <|> SD.Numeric (Const unit) <$ PC.try parseNumericTemplate
         <|> SD.PlainText (Const unit) <$ parsePlainTextTemplate
```

Comment:
I wasn't crazy about this approach - I tried doing something where the time template returned whether seconds were included or not, but I couldn't get the parsing to behave correctly when attempting to do it that way - so we have this thing where seconds are tried explicitly instead.